gulpfile.js

```diff
@@ -60,15 +60,15 @@ gulp.task('influicons', function() {
           fontPath:  'infl-fonts/',
           className: 'ic'
         }))
-        .pipe(gulp.dest('src/icon/'));
+        .pipe(gulp.dest('infl-styles/'));
 
       // generate the patternity readme file for icon
-      gulp.src('infl-icons/templates/icon.readme.md')
-        .pipe(consolidate('lodash', {
-          glyphs:    codepoints
-        }))
-        .pipe(rename('Readme.md'))
-        .pipe(gulp.dest('src/icon/'));
+        // gulp.src('infl-icons/templates/icon.readme.md')
```

Comment:
Why are we commenting this out? It's how patternlab knows what to display for icons.